signing-and-verification-workflow.md

```diff
@@ -0,0 +1,52 @@
+# Signing and Verification Workflow
+
+This document describes how Notary v2 signs and verifies OCI artifacts.
+
+## Signing workflow
+
+The user wants to sign a locally available OCI artifact and push it to a repository(along with signature).
```

Comment:
I think we can remove "locally" as this is one option. I realize this has evolved a bit from the original requirements.
For example, the artifact can be pushed to the registry, and a signature is subsequently pushed to the registry. 
The sign local is an option, but not required.

Reply:
updated to remove locally.